internal/tui/styles/charmtone.go

```diff
@@ -56,6 +56,7 @@ func NewCharmtoneTheme() *Theme {
 		RedDark:  charmtone.Sriracha,
 		RedLight: charmtone.Salmon,
 		Cherry:   charmtone.Cherry,
+		Awaiting:  charmtone.Paprika,
```

Comment:
to match the others let's just call it Paprika

Reply:
If you look a couple more line above, than what GitHub shows you by default, you will find domain named colours.
I think they are better.

<img width="690" height="705" alt="Screenshot 2025-11-12 at 14 11 57" src="https://github.com/user-attachments/assets/7c0164de-e7f8-46b1-959a-3c5f6d2de031" />

Reply:
I am happy to move it to the `// Status` group though.